vendor/engines/saml_authentication/spec/controllers/saml_authentication/sessions_controller_spec.rb

```diff
@@ -11,7 +11,10 @@ def self.decode(saml)
     end
 
     def self.encode(saml)
-      new.send(:encode_raw_saml, saml, OpenStruct.new(compress_request: true))
+      encoded = new.send(:encode_raw_saml, saml, OpenStruct.new(compress_request: false))
```

Comment:
Why does `compress_request` need to change the value?

Reply:
I forgot to change it back after testing